config/module.config.php

```diff
@@ -50,6 +50,19 @@
     'zf-content-validation' => [
         'methods_without_bodies' => [],
         /*
+         * Use this config to validate parameters from routes
+         *
+         * Only configured routes and parameters will validated
+         *
+         * Format follows:
+         *
+         * '<route name>' => [
+         *  '<route.param.1>',
+         *  '<route.param.2>'
+         * ]
```

Comment:
@weierophinney it is just an array describing which params from the route to inject into the InputFilter like so:

```
'user_contact' => [
    'user_id',
    'contact_id',
]
```

This means that for the route named `user_contact` add the params `user_id` and `contact_id` into the InputFilter